CoverMyDotNet/Client.cs

```diff
@@ -56,14 +57,50 @@ public void UpdateRequest(string requestId, string tokenId, string memo)
 			Execute(request);
 		}
 
-		public void DeleteResponse(string requestId, string tokenId, RemoteUserAttributes remoteUser)
+		public IRestResponse DeleteResponse(string requestId, string tokenId, RemoteUserAttributes remoteUser)
 		{
 			var request = new Requests.DeleteRequest(_apiId, tokenId, requestId, new DeleteRequestModel()
 			{
 				RemoteUser = remoteUser,
 				TokenId = tokenId
 			});
-			Execute(request);
+			return Execute(request);
+		}
+
+		public IRestResponse<RequestPageAttributes> GetRequestPage(string requestId, string tokenId)
+		{
+			var request = new Requests.GetRequestPage(_apiId, _apiSecret, requestId, tokenId);
+			return Execute<RequestPageAttributes>(request);
+		}
+
+		public DrugListAttributes SearchDrugs(string query)
+		{
+			var request = new Requests.SearchDrugs(_apiId, query);
+			return Execute<DrugListAttributes>(request).Data;
+		}
+
+		public DrugAttributes GetDrug(string id)
+		{
+			var request = new Requests.GetDrug(_apiId, id);
+			return Execute<DrugAttributes>(request).Data;
+		}
+
+		public FormAttributes GetForm(string id)
+		{
+			var request = new Requests.GetForm(_apiId, id);
+			return Execute<FormAttributes>(request).Data;
+		}
+
+		public FormAttributeList SearchForms(string drugId, string state, string threshold, 
+			 string bin, string pcn, string groupId)
+		{
+			var request = new Requests.SearchForm(_apiId, drugId, state, threshold, bin, pcn, groupId);
+			return Execute<FormAttributeList>(request).Data;
+		}
+			public FormAttributeList SearchForms(string drugId, string state, string q)
```

Comment:
From the outside looking at the API, our users will see a class called "Client" with many methods in it, and I am concerned that it might become challenging to navigate/understand, especially as our APIs evolve and grow.  Would it be worthwhile to break this down into related method groups, perhaps even following the API boundaries?  (drugs, forms, requests, etc)  

cc: @brandonjoyce for a second opinion